docs/basics/basic_httprequest.md

```diff
@@ -0,0 +1,67 @@
+# Steps
+
+Steps are similar to statements in programming language.  An exhaustive list of steps can be found in the [scenario]({{ "/docs/steps/index.html" | absolute_url }}).
+
+Example: 
+
+```yaml
+- thinkTime:
+    duration: 3s
+```
+
+```yaml
+- set: isOneWay <- false
+```
+
+```yaml
+- httpRequest: ...
+```
+
+
+# HTTP Request
+
+One special step is called httpRequest.  httpRequest consists of method (GET, POST, PUT, DELETE etc) and properties.  Properties can be headers and body.  httpRequest API can be found in the [pttpRequest reference](0.6_steps_httpRequest.html).
+
+
+```yaml
+- httpRequest:
+    METHOD:
```

Comment:
You can use the inline form `METHOD: URL_PATH`.

Reply:
oh, I didn't know. let me change it